### ISSUE TITLE
v0.8.0 render error

### ISSUE THREAD

**Issue by imgss** [antfu/wenyan-lang-vscode#7]
when I click the camera icon, got this:
```
Error: Cannot find module '/Users/imgss/.vscode/extensions/antfu.wenyan-lang-0.8.0/node_modules/@wenyanlang/cli/index.min.js'
```
v0.7.2 test OK

**Comment by antfu** [comment]
Thanks for the feedback! It should be fixed in v0.8.2. Please check it out!

**Comment by imgss** [comment]
👍 👍 👍
